scripts/sct_label_vertebrae.py

```diff
@@ -435,7 +435,17 @@ def main(args=None):
 
     # label discs
     printv('\nLabel discs...', verbose)
-    label_discs('segmentation_labeled.nii', verbose=verbose)
+    printv('\nUn-straighten labeling...', verbose)
```

Comment:
i would remove the line that says `    printv('\nLabel discs...', verbose)` and replace this one:
```suggestion
    printv('\nUn-straighten labeled discs...', verbose)
```